sda/cmd/api/api.go

```diff
@@ -584,3 +585,43 @@ func deprecateC4ghHash(c *gin.Context) {
 		return
 	}
 }
+
+func reVerify(c *gin.Context) {
+	accessionID := strings.TrimPrefix(c.Param("accession"), "/")
+
+	reVerify, err := Conf.API.DB.GetReVerificationData(accessionID)
+	if err != nil {
+		if strings.Contains(err.Error(), "sql: no rows in result set") {
+			c.AbortWithStatusJSON(http.StatusNotFound, "accession ID not found")
+		} else {
+			log.Errorln("failed to get file data")
+			c.AbortWithStatusJSON(http.StatusInternalServerError, err.Error())
+		}
+
+		return
+	}
+	corrID, err := Conf.API.DB.GetCorrID(reVerify.User, reVerify.FilePath)
+	if err != nil {
+		log.Errorf("failed to get CorrID for %s, %s", reVerify.User, reVerify.FilePath)
+		c.AbortWithStatusJSON(http.StatusInternalServerError, err.Error())
+
+		return
+	}
+
+	marshaledMsg, _ := json.Marshal(&reVerify)
```

Comment:
Maybe a naive question, why do you ignore the error message from `json.Marshal`? I find that they have been ignored at almost all other places.